CI/check_go.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+THIS_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)
```

Comment:
You may want to add quote to be safe: `THIS_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"`